src/styles/main.scss

```diff
@@ -0,0 +1,6 @@
+@import "_vars.scss";
+@import "clock.scss";
+
+body {
+  margin: 0;
+}
```

Comment:
mentors will ask to move body to reset.scss

Reply:
yeah, and now I ask you to move it to `reset.scss`)